did-utils/src/methods/did_peer/mod.rs

```diff
@@ -1,12 +1,4 @@
-//! The did:peer DID method is designed to be used independent of any central source of truth,
-//! and is intended to be cheap, fast, scalable, and secure. It is suitable for most private
-//! relationships between people, organizations, and things.
```

Comment:
I do not see where you moved this comment.

Reply:
The `did_peer` module is now private. This is an API documentation, not a specification document. Users who want to use Did Peer already know it and where to find its documentation I think. We can include a brief overview of the methods implemented and a short description of them in the `methods` module-level documentation too.

Reply:
This aimed to be a such short description of the `did:peer` method. Can you point me to a suitable replacement in the module-level documentation you added?

Reply:
Already address this in the doc ticket